### PR TITLE
Add help buttons to the masking dialogs

### PR DESCRIPTION
**Pull request by bnmajor**
Depends on HEXRD/hexrdgui-docs#9

### CODE REVIEW

File: hexrdgui/mask_manager_dialog.py [hunk 1]
```diff
@@ -37,6 +38,10 @@ def __init__(self, parent=None):
         self.ui = loader.load_file('mask_manager_dialog.ui', parent)
         flags = self.ui.windowFlags()
         self.ui.setWindowFlags(flags | Qt.Tool)
+
+        add_help_url(self.ui.button_box,
+                     'configuration/masking/#mask-manager')
```
Comment (psavery) [inline hexrdgui/mask_manager_dialog.py:43]:
```suggestion
                     'configuration/masking/#managing-masks')
```

File: hexrdgui/mask_regions_dialog.py [hunk 2]
```diff
@@ -37,6 +38,9 @@ def __init__(self, parent=None):
         flags = self.ui.windowFlags()
         self.ui.setWindowFlags(flags | Qt.Tool)
 
+        add_help_url(self.ui.button_box,
+                     'configuration/masking/#ellipse-and-rectange')
```
Comment (psavery) [inline hexrdgui/mask_regions_dialog.py:42]:
```suggestion
                     'configuration/masking/#ellipse-and-rectangle')
```

**Comment by psavery** [comment]
This will be ready to merge after making the changes.

File: hexrdgui/mask_manager_dialog.py (hunk 1 above)
Comment (bnmajor) [reply hexrdgui/mask_manager_dialog.py]: Thanks! Fixed.

File: hexrdgui/mask_regions_dialog.py (hunk 2 above)
Comment (bnmajor) [reply hexrdgui/mask_regions_dialog.py]: Fixed!

**Comment by bnmajor** [comment]
> This will be ready to merge after making the changes.

Changes made :+1:

**Review by psavery — APPROVED** [review]

**Pull request merged**